webapp/sources/api-doc/components/schemas/node-add.yml

```diff
@@ -0,0 +1,100 @@
+# SPDX-License-Identifier: CC-BY-SA-2.0
+# SPDX-FileCopyrightText: 2013-2020 Normation SAS
+type: array
+items:
+  type: object
+  required:
+    - id
+    - hostname
+    - status
+    - os
+  #  - policyServerId [Optional, by default "root"]
```

Comment:
why is it commented?

Reply:
I thought these were required fields, but there are optional so I comment them for better understanding, to keep track